NEWS

```diff
@@ -1,3 +1,27 @@
+05-May-2020 Ignition 2.3.0
+
+  Features:
+
+    - Allow specifying HTTP headers when fetching remote resources (3.1.0)
```

Comment:
Does `Allow specification of HTTP headers when...` read better?

Reply:
Probably, but it also pushes the line to exactly 80 characters. :confused: